pkg/guacrest/openapi.yaml

```diff
@@ -72,34 +141,46 @@ paths:
           $ref: "#/components/responses/InternalServerError"
         "502":
           $ref: "#/components/responses/BadGateway"
-  "/analysis/dependencies":
+  "/v0/artifact/{digest}/vulns":
     get:
-      summary: Identify the most important dependencies
-      operationId: analyzeDependencies
+      summary: Get vulnerabilities for a specific digest
```

Comment:
nit: I would phrase it as `Get vulnerabilities for an artifact, identified by a digest` (also in `/v0/artifact/{digest}/dependencies`)

Reply:
Rephrased, thanks!